AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt

```diff
@@ -18,6 +18,9 @@ alfastrah.ru,turkishairlines.com,atmos-tokyo.com,radarbox.com,ciceksepeti.com,ra
 !
 !
 !
+insider-gaming.com#$##ig-app-cta { display: none !important; }
```

Comment:
There is already:
https://github.com/AdguardTeam/AdguardFilters/blob/759013bef858ef3cb315031ae7f60c7f5a2778e3/AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt#L96
So I think it's not necessary.